WordPressAuthenticator/Signin/LoginPrologueViewController.swift

```diff
@@ -35,13 +41,23 @@ class LoginPrologueViewController: LoginViewController {
 
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
+        
         configureButtonVC()
         navigationController?.setNavigationBarHidden(true, animated: false)
     }
 
     override func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(animated)
         
+        tracker.set(flow: .prologue)
+        
+        if !Self.prologueFlowTracked {
+            tracker.track(step: .prologue)
+            Self.prologueFlowTracked = true
```

Comment:
```suggestion
            self.prologueFlowTracked = true
```

Reply:
Same as above.

Reply:
I changed the variable to be an instance variable instead. I figure it's more appropriate.